CMakeLists.txt

```diff
@@ -112,64 +112,13 @@ with open(join('${CMAKE_SOURCE_DIR}','qadic', 'CPimport.txt')) as fin:
 )
 configure_file(${CMAKE_BINARY_DIR}/CPimport.h.in ${CMAKE_BINARY_DIR}/CPimport.h COPYONLY)
 
-# Configuration checks
-test_big_endian(HAVE_BIG_ENDIAN)
-
-if(NOT HAVE_BIG_ENDIAN)
-    set(HAVE_LITTLE_ENDIAN ON)
-    set(HAVE_DOUBLE_IEEE_LITTLE_ENDIAN ON)
-else()
-    set(HAVE_DOUBLE_IEEE_BIG_ENDIAN ON)
-endif()
-
 # setup for flint-config.h
-check_include_files(alloca.h HAVE_ALLOCA_H)
-check_include_files(dlfcn.h HAVE_DLFCN_H)
-check_include_files(fcntl.h HAVE_FCNTL_H)
-check_include_files(inttypes.h HAVE_INTTYPES_H)
-check_include_files(locale.h HAVE_LOCALE_H)
-check_include_files(memory.h HAVE_MEMORY_H)
-check_include_files(sstream HAVE_SSTREAM)
-check_include_files(stdint.h HAVE_STDINT_H)
-check_include_files(stdlib.h HAVE_STDLIB_H)
-check_include_files(strings.h HAVE_STRINGS_H)
-check_include_files(string.h HAVE_STRING_H)
-check_include_files(sys/mman.h HAVE_SYS_MMAN_H)
-check_include_files(sys/param.h HAVE_SYS_PARAM_H)
-check_include_files(sys/processor.h HAVE_SYS_PROCESSOR_H)
-check_include_files(sys/resource.h HAVE_SYS_RESOURCE_H)
-check_include_files(sys/stat.h HAVE_SYS_STAT_H)
-check_include_files(sys/sysctl.h HAVE_SYS_SYSCTL_H)
-check_include_files(sys/syssgi.h HAVE_SYS_SYSSGI_H)
-check_include_files(sys/systemcfg.h HAVE_SYS_SYSTEMCFG_H)
-check_include_files(sys/times.h HAVE_SYS_TIMES_H)
-check_include_files(sys/time.h HAVE_SYS_TIME_H)
-check_include_files(sys/types.h HAVE_SYS_TYPES_H)
-check_include_files(unistd.h HAVE_UNISTD_H)
-check_include_files(dlfcn.h HAVE_DLFCN_H)
-check_include_files(fpu_control.h HAVE_FPU_CONTROL_H)
-
-
-check_function_exists(alloca HAVE_ALLOCA)
```

Comment:
On MS cl, aka MS Visual C, `alloca` is called `_alloca`, and its prototype is in `malloc.h`.